lib/ansible/cli/__init__.py

```diff
@@ -553,39 +552,10 @@ def get_host_list(inventory, subset, pattern='all'):
 
     @staticmethod
     def get_password_from_file(pwd_file):
+        file_vault_secret = get_file_vault_secret(filename=pwd_file)
+        file_vault_secret.load()
 
-        b_pwd_file = to_bytes(pwd_file)
-        secret = None
-        if b_pwd_file == b'-':
```

Comment:
Why is support for `-` being removed?

Doing so doesn't seem to be required to implement support for symbolic links.

Reply:
Good call. That was an oversight. I reverted this change. (See below.)